File: Lua_5.4/2936 - Quanta Mandioca.lua
--- author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
--- copyright: Josélio Júnior (Lunatic Fox) 2021

local v = 0
local t = { 300, 1500, 600, 1000, 150 }
for i = 1, #t, 1 do v = v + (t[i] * io.read()) end
print(v + 225)

File: JavaScript_12.18/2936 - Quanta Mandioca.js
/**
 * @author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
 * @copyright Josélio Júnior (Lunatic Fox) 2021
*/

const p = require('fs').readFileSync('/dev/stdin', 'utf8'),
    l = p.split('\n');

let i = 0, arr = [];
while (i < 5) arr.push(l[i]), i++;

console.log([300, 1500, 600, 1000, 150]
    .map((x, y) => x * arr[y])
    .reduce((a, b) => a + b) + 225
);